patches/chromium/skip_atk_toolchain_check.patch

```diff
@@ -0,0 +1,35 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: John Kleinschmidt <jkleinsc@github.com>
+Date: Wed, 15 Jul 2020 17:52:32 -0400
+Subject: skip atk toolchain check
+
+this patch is needed to work around the following gn error on
+Linux arm/arm64:
+
+ERROR at //build/config/linux/atk/BUILD.gn:13:1: Assertion failed.
+
+assert(current_toolchain == default_toolchain)
+
+^-----
+
+See //ui/accessibility/BUILD.gn:297:20: which caused the file to be included.
+
+      configs += [ "//build/config/linux/atk" ]
+
+                   ^-------------------------
+"//build/config/linux/atk" is only needed for accessibility unit tests
```

Comment:
Has this been reported in upstream ? they would welcome patch that fixes build for cross-compiling environments.